renderer/src/features/mcp-servers/lib/form-schema-run-mcp-server-with-command.ts

```diff
@@ -41,32 +41,20 @@ const getCommonFields = (workloads: CoreWorkload[]) =>
       })
       .array(),
     networkIsolation: z.boolean(),
-    allowedHosts: z.array(
-      z.object({
-        value: z.string().refine(
-          (val) => {
-            if (val.trim() === '') return true
-            return /^\.?([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$/.test(val)
-          },
-          {
-            message: 'Invalid host format',
-          }
-        ),
-      })
-    ),
-    allowedPorts: z.array(
-      z.object({
-        value: z.string().refine(
-          (val) => {
-            const num = parseInt(val, 10)
-            return !isNaN(num) && num >= 1 && num <= 65535
-          },
-          {
-            message: 'Port must be a number between 1 and 65535',
-          }
-        ),
-      })
-    ),
+    allowedHosts: z
+      .array(
+        z.object({
+          value: z.string(),
+        })
+      )
+      .optional(),
+    allowedPorts: z
+      .array(
+        z.object({
+          value: z.string(),
+        })
+      )
+      .optional(),
```

Comment:
are we passing those in the payload even if networkIsolation is disabled, is that why they’re defined in baseFields in any case?